R/hypothesize.R

```diff
@@ -67,11 +66,19 @@ parse_params <- function(dots, x) {
   }
 
   # add in 1 - p if it's missing
-  if (length(dots[[p_ind]]) == 1) {
+  if(length(p_ind)){
```

Comment:
What does this double if statement do? I'm inclined to not throw a warning for this: I don't think we want students in the habit of doing both p and q since the stat (or CI) is just on p.